code/null-spam-protector/NullSpamProtector.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace SilverStripe\FrameworkTest\NullSpamProtector;
+
+use SilverStripe\Forms\HiddenField;
+use SilverStripe\SpamProtection\SpamProtector;
+
+/**
+ * This is a minimum implementation used in CI for silverstripe/spamprotector so there's a default_spam_protector
+ *
+ * This used to be done by silverstripe/akismet, but that's no longer being used in CMS 5
+ */
+class NullSpamProtector implements SpamProtector
```

Comment:
I think we need to have a `if (class_exists)` here so we can include frameworktest in contexts where the spamprotector module isn't used.

Reply:
Updated